### DISCUSSION TITLE
fail to enter container bash in cloud shell

### DISCUSSION THREAD

**Discussion by youzipi** [kubevela/kubevela#6243] [Q&A]
fail to enter container bash in cloud shell.

fail message: 
```bash
bash-5.1# vela exec xxx -n xxx --component xxx --cluster xxx --pod xxx --container xxx -- bash
Error: applicationresourcetrackers.prism.oam.dev is forbidden: User "xxx" cannot list resource "applicationresourcetrackers" in API group "prism.oam.dev" in the namespace "xxx"
```

vela prism is enabled.

---
version: 
```
bash-5.1# vela version
CLI Version: 1.7.2
Core Version: 
GitRevision: git-4b88cd20
GolangVersion: go1.19.5
```

**Comment by FogDong** [comment]
The kubeconfig you use does not have permission to list applicationresourcetrackers, you can add the auth and try again.

**Reply by lxkaka** [reply]
we use cloudshell in velaux and this kubeconfig file is generated by velaux, this user has app developer permission.
